Gruntfile.js

```diff
@@ -353,12 +359,6 @@ module.exports = function(grunt) {
   }
 
   if (XUNIT) {
-    var eslint = grunt.config('eslint');
-    eslint.options = eslint.options || {};
-    eslint.options.format = 'junit';
-    eslint.options.outputFile = 'reports/eslint.xml';
-    grunt.config('eslint', eslint);
-
     var jscs = grunt.config('jscs');
     jscs.options.reporter = 'junit';
     jscs.options.reporterOutput = 'reports/jscs.xml';
```

Comment:
yes. once the web client stops using the legacy sdk, at least two gruntfiles should go away. I'm pretty sure jscs doesn't get run any more, but since a bunch of stuff should get deleted in the next month or two, i didn't see the need to rock the boat now.